src/main/kotlin/com/faforever/userservice/domain/BanInfo.kt

```diff
@@ -14,21 +14,25 @@ enum class BanLevel {
 }
 
 @Table("ban")
-data class Ban(
+data class BanInfo(
```

Comment:
Why renamed?

Reply:
To make it consistent with how it is named in the api and the java commons

Reply:
na BanInfo is a horrible name and it is due to historic reason. let's keep ban please

Reply:
Got it